server/controllers/sectionController.js

```diff
@@ -43,5 +42,42 @@ class SectionController {
       });
     }
   }
+
+  static async deleteSection(req, res) {
+    if (req.user.isAdmin !== true) {
+      return res.status(401).json({
+        status: 401,
+        error: 'You are not allowed to perform this action',
+      });
+    }
+    try {
+      const { section } = req.params;
+      const sectionInDb = await Section.findOne({
+        where: {
+          sectionName: section.toUpperCase(),
+        },
+      });
```

Comment:
Hi, Good job. 
Anyway, why are you considering ```sectionName``` as primary key to allow admin deleting a section? don't you know that section name could be a kind of sentence? so I guess it that it's very hard to make comparison.

For me, it sounds like you should have used ``` sectionId ``` as unique attribute to reference every book section.

Reply:
the reason why I used the sectionName is that its easy to remember and it's also unique